src/Checkbox.js

```diff
@@ -0,0 +1,59 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import bootstrapUtils, { bsClass } from './utils/bootstrapUtils';
+
+const propTypes = {
+  inline: React.PropTypes.bool,
+  disabled: React.PropTypes.bool,
+};
+
+const defaultProps = {
+  inline: false,
+  disabled: false,
+};
+
+class Checkbox extends React.Component {
+  render() {
+    const {
+      inline, disabled, className, style, children, ...props,
+    } = this.props;
+
+    delete props.bsClass;
```

Comment:
why delete this? Seems like you do it a bunch, it makes sense for components with custom componentClass's but is there another reason I'm missing?